streaming/src/main/scala/org/apache/spark/streaming/DStreamGraph.scala

```diff
@@ -29,8 +29,8 @@ import org.apache.spark.util.Utils
 
 final private[streaming] class DStreamGraph extends Serializable with Logging {
 
-  private val inputStreams = new ArrayBuffer[InputDStream[_]]()
-  private val outputStreams = new ArrayBuffer[DStream[_]]()
+  private var inputStreams = mutable.ArraySeq.empty[InputDStream[_]]
```

Comment:
This no longer has to be a var - can leave it a val

Reply:
if this is a `val`,  which api should use at line 94 & line 101, there is currently a reassignment here

Reply:
Just do as before: `+=`

Reply:
but `+=` is not a member of `mutable.ArraySeq` because `mutable.ArraySeq` is not `Growable ` .....

Reply:
![image](https://user-images.githubusercontent.com/1475305/93906827-3dc78e80-fd2f-11ea-8379-fbb7161f57a7.png)

Reply:
```
Welcome to Scala 2.13.3 (OpenJDK 64-Bit Server VM, Java 1.8.0_232).
Type in expressions for evaluation. Or try :help.

scala> val s = scala.collection.mutable.ArraySeq.empty[Int]
val s: scala.collection.mutable.ArraySeq[Int] = ArraySeq()

scala> s += 1
         ^
       error: value += is not a member of scala.collection.mutable.ArraySeq[Int]
         Expression does not convert to assignment because receiver is not assignable.

```
execute in Scala shell above

Reply:
Oh, ArraySeq can't be appended. OK then I guess anything is fine